### DISCUSSION TITLE
为什么有些链接可以导入，有些不可以？

### DISCUSSION THREAD

**Discussion by earthmoon** [zzzgydi/clash-verge#612] [Q&A]
你好，

在Debian 12系统中，有些机场的订阅链接可以导入，有些不可以。而不可以导入的链接可以在Windows 11系统中导入。 这是什么原因呢？
[Screencast from 2023-06-11 01-39-06.webm](https://github.com/zzzgydi/clash-verge/assets/45897471/e1710c9c-664f-470b-bec9-fb704d36276a)

**Comment by zzzgydi** [comment]
https协议的链接，然后没有域名只有ip可能会有问题吧，要不换成http的试试

**Reply by earthmoon** [reply]
可是机场没有提供http的订阅，且Windows版本不存在这个问题。

**Reply by zzzgydi** [reply]
直接改成http的试试呗

**Comment by 36h** [comment]
请问配置文件不可以拖进去吗？ 配置文件的事在安装的那个目录

**Reply by earthmoon** [reply]
您好，应该不算配置文件吧，是机场订阅链接。

**Reply by zzzgydi** [reply]
如果有现成的配置文件，可以新建一个local的profile，在下面选择文件即可

**Reply by earthmoon** [reply]
我对这个软件的原理不了解，竟然看不懂你说的。
